shanoir-ng-datasets/src/main/java/org/shanoir/ng/solr/model/ShanoirSolrDocument.java

```diff
@@ -0,0 +1,238 @@
+/**
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+/**
+ * NOTE: This class is auto generated by the swagger code generator program (2.2.3).
+ * https://github.com/swagger-api/swagger-codegen
+ * Do not edit the class manually.
+ */
+package org.shanoir.ng.solr.model;
+
+import java.util.Date;
+
+import org.apache.solr.client.solrj.beans.Field;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.solr.core.mapping.Indexed;
+import org.springframework.data.solr.core.mapping.SolrDocument;
+
+/**
+ * @author yyao
+ *
+ */
+@SolrDocument(solrCoreName = "shanoir")
```

Comment:
Is it fixed ? Maybe if we want to index something else later, a more specific name could be interesting

Reply:
And use a constant ?

Reply:
yes this could be a config. it is fixed to 'shanoir' since it is configured like this in the docker-compose file I think